h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -843,6 +843,13 @@ public void cv_mainModelScores(int N, ModelMetrics.MetricBuilder mbs[], ModelBui
     // Now, the main model is complete (has cv metrics)
     DKV.put(mainModel);
   }
+  
+  public boolean isDistributionHuber() {
```

Comment:
can we just keep the one on modelbuilder or model? do we need both?

Reply:
hm yesyes we can have it just on model